### ISSUE TITLE
Error When Starting

### ISSUE THREAD

**Issue by rut5949** [nvisionative/nvQuickSite#364]
When starting, the application runs in the background for a few seconds and then shuts down. I am using version 2.2.0 on Windows 10. The Windows Event Log shows this error:

Application: nvQuickSite.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.FormatException
   at System.Version+VersionResult.SetFailure(ParseFailureKind, System.String)
   at System.Version.TryParseComponent(System.String, System.String, VersionResult ByRef, Int32 ByRef)
   at System.Version.TryParseVersion(System.String, VersionResult ByRef)
   at System.Version.Parse(System.String)
   at System.Version..ctor(System.String)
   at Newtonsoft.Json.Converters.VersionConverter.ReadJson(Newtonsoft.Json.JsonReader, System.Type, System.Object, Newtonsoft.Json.JsonSerializer)

Exception Info: Newtonsoft.Json.JsonSerializationException
   at Newtonsoft.Json.Converters.VersionConverter.ReadJson(Newtonsoft.Json.JsonReader, System.Type, System.Object, Newtonsoft.Json.JsonSerializer)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.DeserializeConvertable(Newtonsoft.Json.JsonConverter, Newtonsoft.Json.JsonReader, System.Type, System.Object)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(Newtonsoft.Json.Serialization.JsonProperty, Newtonsoft.Json.JsonConverter, Newtonsoft.Json.Serialization.JsonContainerContract, Newtonsoft.Json.Serialization.JsonProperty, Newtonsoft.Json.JsonReader, System.Object)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(System.Object, Newtonsoft.Json.JsonReader, Newtonsoft.Json.Serialization.JsonObjectContract, Newtonsoft.Json.Serialization.JsonProperty, System.String)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(Newtonsoft.Json.JsonReader, System.Type, Newtonsoft.Json.Serialization.JsonContract, Newtonsoft.Json.Serialization.JsonProperty, Newtonsoft.Json.Serialization.JsonContainerContract, Newtonsoft.Json.Serialization.JsonProperty, System.Object)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(Newtonsoft.Json.JsonReader, System.Type, Newtonsoft.Json.Serialization.JsonContract, Newtonsoft.Json.Serialization.JsonProperty, Newtonsoft.Json.Serialization.JsonContainerContract, Newtonsoft.Json.Serialization.JsonProperty, System.Object)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(System.Collections.IList, Newtonsoft.Json.JsonReader, Newtonsoft.Json.Serialization.JsonArrayContract, Newtonsoft.Json.Serialization.JsonProperty, System.String)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(Newtonsoft.Json.JsonReader, System.Type, Newtonsoft.Json.Serialization.JsonContract, Newtonsoft.Json.Serialization.JsonProperty, System.Object, System.String)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(Newtonsoft.Json.JsonReader, System.Type, Newtonsoft.Json.Serialization.JsonContract, Newtonsoft.Json.Serialization.JsonProperty, Newtonsoft.Json.Serialization.JsonContainerContract, Newtonsoft.Json.Serialization.JsonProperty, System.Object)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(Newtonsoft.Json.JsonReader, System.Type, Boolean)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(Newtonsoft.Json.JsonReader, System.Type)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(System.String, System.Type, Newtonsoft.Json.JsonSerializerSettings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.String, Newtonsoft.Json.JsonSerializerSettings)
   at nvQuickSite.Controllers.PackageController.GetLocalPackages()
   at nvQuickSite.Controllers.PackageController.GetPackageList()
   at nvQuickSite.Start.LoadPackages()
   at nvQuickSite.Start..ctor(Serilog.Core.LoggingLevelSwitch)
   at nvQuickSite.Main..ctor(Serilog.Core.LoggingLevelSwitch)
   at nvQuickSite.Program.Main()

**Comment by david-poindexter** [comment]
@rut5949 thanks for letting us know about this.  This is the first time we are hearing of the app crashing on load like this.  Do you have the .NET Framework 4.5 runtime installed?

**Comment by rut5949** [comment]
I have 4.8 installed.

**Comment by david-poindexter** [comment]
@rut5949 I have been running Windows 11 for quite some time now and do not have a Windows 10 environment to even test this in.  Perhaps someone else in the community does.

**Comment by rut5949** [comment]
I figured out a work around to get the program to run, but I do not understand what is causing the problem. If I have windows run the compatibility checker for the program it switches the compatibility mode to Windows 8. If I then use the "Test the program" button under "Test compatibility settings for the program" nvQuickSite runs.  If I save the compatibility settings and then try running the program like normal, it fails. 

I also installed nvQuickSite on another Windows 10 PC with the same Windows updates and it worked on that PC. So this is an issue with my PC and I have a work around. If I do figure out what is causing the issue, I will post it hear.
